base_comment_template/migrations/16.0.1.1.0/pre-migration.py

```diff
@@ -8,20 +8,21 @@
 
 @openupgrade.migrate()
 def migrate(env, version):
-    if not column_exists(env.cr, "base_comment_template", "models"):
-        openupgrade.logged_query(
-            env.cr,
-            "ALTER TABLE base_comment_template ADD COLUMN IF NOT EXISTS models text",
-        )
-        openupgrade.logged_query(
-            env.cr,
-            """
-            UPDATE base_comment_template template
-            SET models = (
-                SELECT string_agg(model.model, ',')
-                FROM base_comment_template_ir_model_rel AS rel
-                JOIN ir_model AS model ON rel.ir_model_id = model.id
-                WHERE rel.base_comment_template_id = template.id
+    if openupgrade.table_exists(env.cr, "base_comment_template_ir_model_rel"):
+        if not column_exists(env.cr, "base_comment_template", "models"):
```

Comment:
Please reduce diff and complexity with this small change:

```suggestion
    if (openupgrade.table_exists(env.cr, "base_comment_template_ir_model_rel") and
        not column_exists(env.cr, "base_comment_template", "models")):
```